src/DetectChanges/Variance/TypeIsContravariant.php

```diff
@@ -29,12 +35,17 @@ public function __invoke(
             return false;
         }
 
+        if (! $type instanceof ReflectionNamedType || ! $comparedType instanceof ReflectionNamedType) {
+            // @TODO we'll assume everyting is fine, for now - union and intersection types still need test additions
```

Comment:
This will be picked up by `infection/infection`, but basically, I did **not** test beyond `ReflectionNamedType` for now.